src/libse/NetflixQualityCheck/NetflixQualityController.cs

```diff
@@ -54,6 +121,41 @@ public int SingleLineMaxLength
             }
         }
 
+        public DialogType SpeakerStyle
+        {
+            get
+            {
+                switch (Language)
+                {
+                    case "ar": // Arabic
+                    case "cs": // Czech
+                    case "fr": // French
+                    case "hu": // Hungarian
+                    case "in": // Indonesian
+                    case "it": // Italian
+                    case "ko": // Korean
+                    case "ms": // Malay
+                    case "pl": // Polish
+                    case "ro": // Romanian
+                    case "ru": // Russian
+                    case "sk": // Slovak
+                    case "es": // Spanish
+                    case "th": // Thai
+                    case "vi": // Vietnamese
+                        return DialogType.DashBothLinesWithSpace;
+                    case "nl": // Dutch
+                    case "fo": // Finnish
```

Comment:
Shouldn't this be `fi`?

Reply:
Oh, yeah. I got confused and took the first result:
![image](https://user-images.githubusercontent.com/20923700/104229607-2b47a100-5455-11eb-98db-8b03b86fab07.png)

I mean, I just want Finnish, why show me the rest? xD